packages/formvuelate/src/utils/Helpers.js

```diff
@@ -69,3 +86,33 @@ export const forEachSchemaElement = (schema, fn, path = '') => {
     }
   }
 }
+
+/**
+ * https://stackoverflow.com/questions/6393943/convert-a-javascript-string-in-dot-notation-into-an-object-reference
+ * @param {String} path - The path plus the property ex. "nested.firstName" NOT "nested" by itself
+ * @param {Object} object
+ * @returns {*} property value
+ */
+export const findPropertyForPath = (path, object) => {
+  return path.split('.').reduce((step, index) => step[index], object)
+}
+
+/**
+ * Execute a user defined function for each element in a form model object
+ * @param {Object|Ref} formModel
+ * @param {Function} fn
+ * @param {String} path
+ */
+export const forEachPropInModel = (formModel, fn, path = '') => {
+  const rawModel = unref(formModel)
+
+  for (const prop in rawModel) {
+    const value = rawModel[prop]
+
+    if (typeof value === 'object') {
```

Comment:
I assume this could crash in case of `null`?

Reply:
Good point, ill make a double check